src/core/storage/sqliteStorage/migrations/version-0.0.1.ts

```diff
@@ -12,7 +12,7 @@ const DATA_V001 = {
             id INTEGER PRIMARY KEY AUTOINCREMENT,
             item_id TEXT NOT NULL,
             name TEXT NOT NULL,
-            value TEXT NOT NULL,
+            value TEXT,
```

Comment:
Hmm, I don't think this should be changing, right?

The idea I had was if the tag is undefined - for SQL, don't insert it. And when searching, if the tag doesn't exist - it is the same as being undefined. I would rather not insert null values if we don't have to.

Reply:
Hmm... that is opposite with the discussing before. That mean we will searching with not exist if it is underfined, null, "", right?

Reply:
When we insert to DB with "", I remembered it is null

Reply:
Let me test it.